index.html

```diff
@@ -8,6 +8,7 @@
 
   <body>
     <div id="mount"></div>
+    <script>window.debuggerConfig = <%= debuggerConfig %>; </script>
```

Comment:
I was thinking we could do something with webpack.production.config to inject config. Although... we could probably do that with webpack.config as well